compaction_picker.go

```diff
@@ -1586,6 +1586,42 @@ func (nml NoMultiLevel) pick(
 	return pc
 }
 
+func (pc *pickedCompaction) predictedWriteAmp() float64 {
```

Comment:
when evaluating the size of the files to compact,  I wonder if I should I create a new `files.SizeSumWithoutTombstones()` which subtracts each file's `file.stats.PointDeletionsBytesEstimate` or `file.stats.RangeDeletionsBytesEstimate`?